airgun/views/common.py

```diff
@@ -80,6 +81,35 @@ def read(self, widget_names=None, limit=None):
                 values[widget_name] = widget.read()
         return normalize_dict_values(values)
 
+    def documentation_links(self):
+        """Return Documentation links present on the given page if any.
+        Note: This is not a full-proof helper. For example, it can't get links hidden behind a dropdown button.
```

Comment:
I would say, let's add that feature of getting the link from the dropdowns later. Right? :D

Reply:
yes, I'd leave it for later too. One problem at a time:)

Reply:
Agree